website/blog/2024-10-05-kubernetes-blog.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Build your Kubernetes application with Podman Desktop
+description: Covers the end-to-end workflow to create a Kubernetes application
+authors: [shipsing]
+tags: [podman-desktop, podman, Kubernetes-application, deployment]
+hide_table_of_contents: false
+---
+
+
+# Using Podman Desktop to create a Kubernetes application
+
+The integration of Podman Desktop with Kubernetes helps you to run your application on a Kubernetes cluster, such as Kind or Minikube.
+
+This blog covers the following aspects:
+- Build a containerized application from registry images
+- Create a pod
+- Set up a local Kubernetes cluster
+- Deploy the application to Kubernetes
+- Verify the running
+ service
+
+## Building a containerized application 
+
+You can build a containerized application that uses:
+- a frontend Python application container
+- a backend Redis server container 
```

Comment:
Below, you are describing how to pull and start the Redis server first. Perhaps the 2 containers should be ordered differently here, with the Redis container first?